### DISCUSSION TITLE
Can I supply a list of URLs to check?

### DISCUSSION THREAD

**Discussion by ms2art** [linkchecker/linkchecker#727] [Q&A] [answered]
I'd like to pass a file, containing a list of URLs to check, as an argument. This looks like the way to do it:

FILE-OR-URL[](https://linkchecker.github.io/linkchecker/man/linkchecker.html#cmdoption-arg-FILE-OR-URL)

    The location to start checking with. A file can be a simple list of URLs, one per line, if the first line is “# LinkChecker URL list”.

I don't know the right way to specify that argument on the command line, though. Can someone provide a sample command that uses FILE-OR-URL?

**Comment by cjmayo** [comment]
For example:
`linkchecker myurls.txt`

It's just the most common argument specifying the location to be checked.

**Comment by ms2art** [comment]
Thanks. And I should have asked, what should the content of myurls.txt look like?

**Comment by cjmayo** [comment, marked as answer]
As you copied:
> a simple list of URLs, one per line, if the first line is “# LinkChecker URL list”

Just try it!
